project1/model.py

```diff
@@ -1,10 +1,44 @@
 from flask_sqlalchemy import SQLAlchemy
 
+
 db = SQLAlchemy()
 
 class Registartion(db.Model):
     __tablename__ = "Registartion"
     Firstname = db.Column(db.String, primary_key=True)
     Email = db.Column(db.String, nullable=False)
```

Comment:
During registration are you not storing password?Username and password,date time are the mandatory fields.Discuss with your time and update.